micrometer-core/src/test/java/io/micrometer/core/instrument/binder/cache/HazelcastCacheMetricsTest.java

```diff
@@ -19,18 +19,32 @@
 import com.hazelcast.core.Hazelcast;
 import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.core.IMap;
-import io.micrometer.core.instrument.Statistic;
+import io.micrometer.core.instrument.FunctionTimer;
 import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static java.util.Collections.emptyList;
 import static org.assertj.core.api.Assertions.assertThat;
 
 class HazelcastCacheMetricsTest {
+
+    private HazelcastInstance h;
+
+    @BeforeEach
+    void setup(){
+        Config config = new Config();
+        h = Hazelcast.newHazelcastInstance(config);
+    }
+
+    @AfterEach
+    void cleanup(){
+        h.shutdown();
```

Comment:
Hopefully shutting down hazelcast will help a little with the memory problems we've seen in the test runs. Probably not much...

Reply:
Good call!